src/main/java/org/mskcc/cmo/messaging/NATSConnection.java

```diff
@@ -0,0 +1,22 @@
+package org.mskcc.cmo.messaging;
+
+import io.nats.client.Connection;
+import io.nats.client.Nats;
+import java.io.IOException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class NATSConnection {
+
+    @Value("${nats.connection_url}")
+    private String natsConnectionUrl;
+
```

Comment:
Should nats.flush_duration be captured here?

Reply:
I dont think so, this is just where the connection is established whereas the flush duration is a param that is used by an established connection like in the NATSGatewayImpl